.github/workflows/issue-handling.yaml

```diff
@@ -29,6 +29,25 @@ jobs:
           github-token: ${{ secrets.ORG_AUTOMATION_TOKEN }}
           labeled: continuous_aggregate
 
+  notify-sec:
+    name: Notify security channel
+    runs-on: ubuntu-latest
+    if: github.event_name == 'issues' && contains(github.event.issues.issue.labels.*.name, 'segfault')
```

Comment:
should probably also trigger on security

Reply:
Updated.